README.md

```diff
@@ -4,13 +4,14 @@ Tiny option parser for [mruby][mruby] with __zero__ dependencies.
 
 ```ruby
 parser = OptParser.new do |opts|
-  opts.on(:port, :int, 80) { |port| ... }
-  opts.on(:ip, :string, '127.0.0.1') { |ip| ... }
+  opts.on(:port, :int, default: 80) { |port| ... }
```

Comment:
you forget to update the doc :)

Reply:
This has now been resolved, thank you for your patience!